### PR TITLE
1.1.1 bugfix release

### PR DESCRIPTION
**Pull request by asmeurer**
Looks like a lot of little issues have popped up with 1.1, so I'm going a 1.1.1 bugfix release. 

This branch is based on the 1.1 branch, so if you want any fixes here, please be sure they are made against this branch, not master.

### CODE REVIEW

File: sympy/core/power.py [hunk 1]
```diff
@@ -1062,15 +1062,22 @@ def _eval_is_rational(self):
...
+                return (expr - 1).is_zero
+            except ValueError:
+                # when the operation is not allowed
+                return False
+
+        if self.base.is_zero or _is_one(self.base):
             return True
         elif self.exp.is_rational:
             if self.base.is_algebraic is False:
                 return self.exp.is_nonzero
```
Comment (jksuom) [inline sympy/core/power.py]: This should be `is_zero`.

Comment (asmeurer) [inline sympy/core/power.py]: I believe this is unrelated to the above fix (let me know if I am wrong). If so, we should fix this in a PR to master.

**Comment by jirikuncar** [comment]
Please merge regularly maintenance branches to master:

```
master
------
o (v1.1.0)
| \
o   o maint-1.1
|   | ---------
o   o
| / |  <- regular merge
o   o (v1.1.1)
| / |  <- regular merge
o   o
|   |
o   o (v1.1.2)
| / |
o   .
.   .
.   .

**Comment by asmeurer** [comment]
Sorry, I got caught up with SciPy. I am hoping to finish this up this week.

**Comment by jirikuncar** [comment]
> Sorry, I got caught up with SciPy. I am hoping to finish this up this week.

The release is not so important as long as all commits from 1.1.1 are immediately merged to master so we don't have to manually merge or cherry-picked fixes which appear in both branches.

The releases should happen independently on merges between maintenance branches. See a real life example from another project:

```
*   3e246af0e Merge branch 'maint-1.2' into legacy
|\
| *   48c62330b Merge branch 'maint-1.1' into maint-1.2
| |\
| | *   89b4eee2a Merge branch 'maint-1.0' into maint-1.1
| | |\
| | | * 75373d2c2 installation: support for Ubuntu 14.04 LTS
| | | * cc5779c16 installation: use of `/usr/sbin/service`
| | | * 4585ef594 WebSession: silencing MySQL UTF-8 warnings
| * | | 5a7481ac7 installation: Apache and Redis start-up
| * | | 6a9cf772f WebSearch: fix for da today 3650 test case
| * | | 1c2818e96 BibCatalog: no email tests when no SMTP server
| * | | 23c9c67dc installation: web node h5py provisioning
* | | |   cebdae899 Merge branch 'maint-1.2' into legacy
|\ \ \ \
| |/ / /
| * | |   7e02ddc5f Merge branch 'maint-1.1' into maint-1.2
| |\ \ \
| | |/ /
| | * | d8fc81d9b installation: Invenio hosting of jQuery plugins
| | * |   aa0513494 Merge branch 'maint-1.0' into maint-1.1
| | |\ \
```
